Sources/SKSwiftPMWorkspace/SwiftPMWorkspace.swift

```diff
@@ -205,6 +205,11 @@ extension SwiftPMWorkspace {
   /// dependencies.
   func reloadPackage() async throws {
     await reloadPackageStatusCallback(.start)
+    defer {
+      Task {
+        await reloadPackageStatusCallback(.end)
```

Comment:
Seems a bit weird for the end callback to be run at some point and not before `reloadPackages` returns. Could we sink what's in `reloadPackage` down into another function then call it here wrapped in `do` to make sure we always call end?

Reply:
Discussed offline and decided that it’s OK.